src/timelines.js

```diff
@@ -85,7 +76,7 @@ export default Kapsule({
                   label: rawData[i].data[j].label,
                   timeRange: (dateObjs
                       ?rawData[i].data[j].data[k].timeRange
-                      :[new Date(rawData[i].data[j].data[k].timeRange[0]), new Date(rawData[i].data[j].data[k].timeRange[1])]
+                      :[parseDate(rawData[i].data[j].data[k].timeRange[0]), parseDate(rawData[i].data[j].data[k].timeRange[1])]
```

Comment:
@Steve-Pegg this means that every data input will be parsed with the `timeFormat`. This is an undesired breaking change. As per the spec, the input data should have plain `Date` objects. If the timestamps in your data are in a different format, you should convert them externally, prior to feeding them to the library. The `timeFormat` component prop is only for the labelling purposes.

Reply:
I have added optional date parser.  How does the PR work?  Do you accept or reject proposals?

Reply:
Thanks for updating. I certainly accept and encourage fixes and updates. But I find it's a cleaner approach to not make the data formatting the responsibility of the library, but that of the consumer. It has better separation of concerns.